docs/blogs/EnhancedDevPreviewGatewayAPI/GettingStarted.md

```diff
@@ -0,0 +1,182 @@
+```
+
+Gateway API and OSSM have been installed successfully and are ready to use. We are now ready to create Gateways and HTTPRoutes.
+
+## Gateway API Example
+
+Let’s now use Gateway API to create a Gateway and Route. In this example, we will convert a simple HTTP OpenShift Route into roughly equivalent Gateway API objects (Gateway, HTTPRoute). This is an example of OpenShift Route that we will convert:
+```yaml
+apiVersion: v1
+kind: Route
+metadata:
+  name: http
+  namespace: demo-app
+spec:
+  host: demo.gwapi.${DOMAIN}
+  to:
+    kind: Service
+    name: foo-app
+  port:
+    targetPort: 8080
+```
+
+First, create the Gateway with a Listener (`spec.Listeners` instance) that will listen for requests on a URL under the cluster domain. The Gateway must be created in the `openshift-ingress` namespace and specify in the `spec.gatewayClassName` the GatewayClass we created earlier.
+```yaml
+$ DOMAIN=$(oc get ingresses.config/cluster -o jsonpath={.spec.domain})
+$ oc apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: demo-gateway
+  namespace: openshift-ingress
+spec:
+  gatewayClassName: openshift-default
+  listeners:
+  - name: demo
+    hostname: "*.gwapi.${DOMAIN}"
+    port: 80
+    protocol: HTTP
+    allowedRoutes:
+      namespaces:
+        from: All
+EOF
+```
+
+By default, Istio will [automatically provision](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/#automated-deployment) a gateway deployment and service with the same name upon creation of this Gateway object:
+```shell
+$ oc get deployment -n openshift-ingress demo-gateway
+NAME           READY   UP-TO-DATE   AVAILABLE   AGE
+demo-gateway   1/1     1            1           25s
+$ oc get service -n openshift-ingress demo-gateway
+NAME           TYPE           CLUSTER-IP       EXTERNAL-IP                          PORT(S)                        AGE
+demo-gateway   LoadBalancer   172.30.175.176   domain.us-east-1.elb.amazonaws.com   15021:30608/TCP,80:31833/TCP   47s
+```
+
+When the Gateway from the previous step is created, the Ingress Operator will automatically create a DNSRecord with the hostname from the listener:
+```shell
+$ oc get dnsrecord -n openshift-ingress
+NAME                         AGE
+gateway-7b49f58c6-wildcard   6m11s
+```
+
+Next, let’s create and configure a namespace called demo-app that will have our HTTPRoute object and our example application:
+```shell
+$ oc create namespace demo-app
+```
+
+Create a demo application deployment and service using `oc new-app`, our cakephp-ingress-demo project, and the foo git branch. This will create a simple server, foo-app, that will allow us to test connectivity. Then use the `oc rollout` command to wait for it to be rolled out:
+```shell
+$ oc new-app -n demo-app --name foo-app https://github.com/openshiftdemos/cakephp-ingress-demo#foo
+$ oc rollout status deployment -w -n demo-app foo-app
+```
```

Comment:
Better to write out "the ... command" and "... GitHub project" explicitly.  In the "Getting Creative" section, you use ``` `bar` ``` and ``` `cakephp-ingress-demo` ```; please use consistent formatting here:
````suggestion
Create a demo application deployment and service using the `oc new-app` command and the `foo` branch of our `cakephp-ingress-demo` GitHub project. This will create a simple server, named "foo-app", that will allow us to test connectivity. Then use the `oc rollout` command to wait for it to be rolled out:
```console
$ oc new-app -n demo-app --name foo-app https://github.com/openshiftdemos/cakephp-ingress-demo#foo
$ oc rollout status deployment -w -n demo-app foo-app
[expected output]
```
````
What output is the reader expected to see `oc rollout status` print out?

Reply:
Done and added the expected output.